glide.lock

```diff
@@ -61,7 +61,7 @@ imports:
 - name: github.com/decred/dcrrpcclient
   version: c25ea196a4f6bd9c37a7d74224f6e6027c3ee3d7
 - name: github.com/decred/dcrutil
-  version: ddbde93f65ab0692e54ed8a5ad325fa2e8af4daa
+  version: a5fab53cab39b793142c8453caa4c6f83bc152d4
   subpackages:
```

Comment:
This actually is going backwards.  Glide is kind of messed up as it loves to update everything even when but requested, but also it seems incapable of hosting is cache.  Before a glide update, or apparent glide get, it's necessary to glide cc.  But usually I just pick pieces of glide.lock edits to keep with git add -p glide.lock